atlas/packages/app/src/shared/__tests__/Button.test.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import { mount } from 'enzyme'
-import { Button } from '../components'
+import Button from '@/shared/components/Button'
```

Comment:
This is inside the `shared` package so I think it may be a better idea to use a relative import here